### PR TITLE
Initialize ICML Reproducible Machine Learning workshop

### PR DESCRIPTION
**Pull request by pmandler**

### CODE REVIEW

File: venues/ICML.cc/2017/RML/python/admin-init.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    utils.get_path("../process", __file__)
+    )
+
+groups = {}
+groups[config.PROGRAM_CHAIRS] = openreview.Group(config.PROGRAM_CHAIRS, **config.group_params)
+groups[config.AREA_CHAIRS] = openreview.Group(config.AREA_CHAIRS, **config.group_params)
+groups[config.REVIEWERS] = openreview.Group(config.REVIEWERS, **config.group_params)
+
+invitations = {}
+invitations[config.SUBMISSION] = openreview.Invitation(config.SUBMISSION, duedate=1497733200000, **config.submission_params)
```
Comment (melisabok) [inline venues/ICML.cc/2017/RML/python/admin-init.py]: Based on the duedate timestamp the deadline is June 17th 5 PM, right?

File: venues/ICML.cc/2017/RML/webfield/conf.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+// never changes, put it in its own function
+function renderConferenceHeader() {
+  Webfield.ui.venueHeader({
+    title: "ICML 2017 RML Workshop",
+    subtitle: "Reproducibility in Machine Learning",
+    location: "Syndey, Autralia",
+    date: "June 17th, 2017 5:00pm",
+    website: "https://2017.icml.cc/Conferences/2017/Schedule?showEvent=16",
+    instructions: null,  // Add any custom instructions here. Accepts HTML
+//    deadline: '2017-06-06 17:00:00'
```
Comment (melisabok) [inline venues/ICML.cc/2017/RML/webfield/conf.html]: why the deadline here says June 6th?

Comment (pmandler) [inline venues/ICML.cc/2017/RML/webfield/conf.html]: I'm surprised to see that.  It appears it came from my admin/conference-template file.  It was only supposed to be changed for the RSS conference. I'll look into it.

Comment (pmandler) [inline venues/ICML.cc/2017/RML/webfield/conf.html]: It appears to have snuck in with Michael's changes last night.  I think we want 'date' to be the date of the conference and add deadline for the submission with the text "Submission Deadline: "

**Comment by melisabok** [comment]
Cool, thanks!

**Comment by melisabok** [comment]
I see that value was hardcoded in the template, I'll ask @mspector to fix it. 

https://github.com/iesl/openreview-scripts/blob/master/admin/conference-template/webfield/conf.template#L45

**Comment by pmandler** [comment]
I have a fix, want me to test it out or should I just let Michael take it?

**Comment by melisabok** [comment]
Please go ahead, just assign yourself to this issue: https://github.com/iesl/openreview-scripts/issues/387

**Pull request merged**